src/views/Vote/VoteForm/VoteForm.vue

```diff
@@ -64,6 +59,15 @@ export default class VoteForm extends Vue {
   @Votings.Action("createVote")
   private createVote: any;
 
+  private input = {
+    score: (this.voteScore) ? this.voteScore : 3,
```

Comment:
It seems there're more than one variable/propery/object setting the score value, why not just one. It's hard to follow the logic.